src/widgets/mod.rs

```diff
@@ -132,6 +135,41 @@ pub(crate) fn default_theme() -> Theme {
         })
         .apply(focus_style.clone());
 
+    let radio_button_style = Style::new()
+        .width(20.)
+        .height(20.)
+        .background(Color::WHITE)
+        .active(|s| s.background(active_bg_color))
+        .transition(Background, Transition::linear(0.04))
+        .hover(|s| s.background(hover_bg_color))
+        .focus(|s| s.hover(|s| s.background(focus_hover_bg_color)))
+        .apply(border_style.clone())
```

Comment:
I'd apply a 100% border radius after here.

Reply:
Much better, thank you. The alignment issue is still lurking though:

![image](https://github.com/lapce/floem/assets/1407980/cec157cd-88ad-4d6e-b68e-f50204bd275f)

Reply:
Not quite sure about that. You could try using a centered `empty` instead of an SVG.

Reply:
Interesting, I'll give that a spin.

Reply:
I've rewritten it to use `empty`, but the alignment is still slightly off. I've used `.align_items(AlignItems::CENTER).justify_content()` to center the dot.

![image](https://github.com/lapce/floem/assets/1407980/c65478f9-d0a8-487a-8f8f-ea8d7bbb5b3a)